docs/fs.md

```diff
@@ -6,6 +6,12 @@ directory from the host with the guest.
 __virtio-fs__, also known as __vhost-user-fs__ is a virtual device defined by
 the VIRTIO specification which allows any VMM to perform filesystem sharing.
 
```

Comment:
The commit message summary is too long (72 character limit). This would be better:

```
docs: Add limitations section to virtio-fs documentation

In particular highlight that snapshot/restore and live migration are not 
supported.
```